app/webpacker/components/RegistrationsV2/RegistrationAdministration/RegistrationAdministrationList.jsx

```diff
@@ -180,8 +180,21 @@ export default function RegistrationAdministrationList({ competitionInfo }) {
             return a.competing.registered_on.localeCompare(
               b.competing.registered_on,
             );
-          case 'paid_on':
-            return a.payment.updated_at.localeCompare(b.payment.updated_at);
+          case 'paid_on_with_registered_on_fallback':
+          {
+            if (a.payment && b.payment) {
+              return a.payment.updated_at.localeCompare(b.payment.updated_at);
```

Comment:
Not a fan of comparing dates as strings. If this is currently working as intended, then probably it's only a "happy accident".

You should parse these timestamps into either JS-native `Date` or Luxon `DateTime` representation and then use those to compare